AccountingSystem/private/DBUtils.py

```diff
@@ -314,3 +314,16 @@ def _getBucketTotals( self, dataDict ):
           newData[ bt ] = 0.0
         newData[ bt ] += dataDict[ k ][ bt ]
     return newData
+  
+  def _getOverallMaxValue(self, dataDict):
```

Comment:
I am not quite sure what you are trying to do with this method. But I *think* that 
```max([ x for y in d.itervalues() for x in y.itervalues() ])``` does what you want no ?